jerry-main/main-unix.c

```diff
@@ -121,6 +121,33 @@ print_help (char *name)
                       name);
 } /* print_help */
 
+static void
+print_unhandled_exception (jerry_value_t error_value)
+{
+  assert (jerry_value_has_error_flag (error_value));
+
+  jerry_value_clear_error_flag (&error_value);
+  jerry_value_t err_str_val = jerry_value_to_string (error_value);
+  jerry_size_t err_str_size = jerry_get_string_size (err_str_val);
+  jerry_char_t err_str_buf[256];
+
+  if (err_str_size >= 256)
+  {
+    const char msg[] = "[Long message]";
+    err_str_size = sizeof (msg) / sizeof (char);
```

Comment:
There is nothing wrong with sizeof() / sizeof(), it just have to be used carefully. It still has the advantage over strlen() of being a compile time constant and not requiring any code execution at runtime.